src/commands/compile/spec/compile.spec.ts

```diff
@@ -171,3 +178,209 @@ describe('sasjs compile single file', () => {
     })
   })
 })
+
+const defaultBuildConfig: BuildConfig = {
+  buildOutputFolder: 'sasjsbuild',
+  buildOutputFileName: 'test.sas',
+  initProgram: '',
+  termProgram: '',
+  macroVars: {}
+}
+describe('sasjs compile outside project', () => {
```

Comment:
it would be nice to verify that jobs and services were compiled properly (including all dependencies).

Reply:
tests updated for `compile` and `run`